src/app/records/plugset-helpers.ts

```diff
@@ -1,22 +1,6 @@
 import { DimPlugSet } from 'app/inventory/item-types';
 import { DestinyProfileResponse } from 'bungie-api-ts/destiny2';
 
-/**
- * Get all plugs from the specified plugset. This includes whether the plugs are unlocked or not.
- * This returns the specified plugset no matter where it lives.
- * @deprecated try to ask for a specific character or profile-wide
- */
-export function itemsForPlugSetEverywhere(
-  profileResponse: DestinyProfileResponse,
-  plugSetHash: number
-) {
-  return (profileResponse.profilePlugSets.data?.plugs[plugSetHash] || []).concat(
-    Object.values(profileResponse.characterPlugSets.data || {})
-      .filter((d) => d.plugs?.[plugSetHash])
-      .flatMap((d) => d.plugs[plugSetHash])
-  );
-}
-
```

Comment:
I deleted this as its no longer used but I will need to create another check for the subclass stuff to ensure aspects/fragments are unlocked before I can merge this.